<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editsupplier</class>
 <widget class="QDialog" name="Editsupplier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../icons/resources.qrc">
    <normaloff>:/png/marca-m.png</normaloff>:/png/marca-m.png</iconset>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>761</width>
     <height>350</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>731</width>
      <height>130</height>
     </rect>
    </property>
    <property name="title">
     <string>Telefones</string>
    </property>
    <widget class="QTableView" name="tableViewPhone">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>24</y>
       <width>660</width>
       <height>90</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAddPhone">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>46</y>
       <width>30</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRemovePhone">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>74</y>
       <width>30</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>731</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="LblNome">
       <property name="text">
        <string>Nome</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LnEdtNome"/>
     </item>
     <item>
      <widget class="QLabel" name="LblCNPJ">
       <property name="text">
        <string>CNPJ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LnEdtCNPJ">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>731</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="LblFantasia">
       <property name="text">
        <string>Fantasia</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LnEdtFantasia"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>730</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string> Endere√ßos </string>
    </property>
    <widget class="QTableView" name="tableViewAddress">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>651</width>
       <height>100</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAddAddress">
     <property name="geometry">
      <rect>
       <x>679</x>
       <y>53</y>
       <width>30</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRemoveAddress">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>80</y>
       <width>30</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="topmenu">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>801</width>
     <height>42</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>42</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>42</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/png/barra-superior-ok.jpg);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelStatus">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>0</y>
      <width>541</width>
      <height>39</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Cadastro de Fornecedores</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="PshBtnCancel">
   <property name="geometry">
    <rect>
     <x>599</x>
     <y>408</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PshBtnSave">
   <property name="geometry">
    <rect>
     <x>686</x>
     <y>408</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Salvar</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>LnEdtNome</tabstop>
  <tabstop>LnEdtCNPJ</tabstop>
  <tabstop>LnEdtFantasia</tabstop>
  <tabstop>tableViewPhone</tabstop>
  <tabstop>pushButtonAddPhone</tabstop>
  <tabstop>pushButtonRemovePhone</tabstop>
  <tabstop>PshBtnCancel</tabstop>
  <tabstop>PshBtnSave</tabstop>
 </tabstops>
 <resources>
  <include location="../../icons/resources.qrc"/>
  <include location="../../icons/resources.qrc"/>
 </resources>
 <connections/>
</ui>
